pipelines/powershell-package/default-ci-pipeline.yaml

```diff
@@ -0,0 +1,41 @@
+parameters:
+  excludeRules: [] # List of rules. See https://github.com/PowerShell/PSScriptAnalyzer/tree/master/RuleDocumentation
+  nugetversion: '5.4.0'
+  pathToProjects: '$(Build.SourcesDirectory)/src/'
+  nugetDevOpsFeedToPublish: '/8187f5c9-e9c1-419f-8a5c-98285cf7633c' #GlobalDev
+  pushpackage: 'false'
```

Comment:
if you specify the parameter like this you would be not able to set it during run time. Is this what you want?